src/api/InkAPITest.cc

```diff
@@ -8653,77 +8653,145 @@ EXCLUSIVE_REGRESSION_TEST(SDK_API_TSHttpConnectServerIntercept)(RegressionTest *
 ////////////////////////////////////////////////
 
 // The order of these should be the same as TSOverridableConfigKey
+// clang-format off
```

Comment:
I haven't found good clang-format config to achieve this.

Reply:
What do you think about adding a comment that we did this to reduce diff sizes?

Reply:
I guess, it's obvious :) (I don't object to add comments, though)